### ISSUE TITLE
React v 0.14 warns for using deprecated React.findDOMNode()

### ISSUE THREAD

**Issue by azazdeaz** [azazdeaz/react-gsap-enhancer#8]
I know about this but if i use react-dom, i can't support React v0.13 anymore which most people use at the time.

See: facebook/react#5155

But i going to release the react-gsap-enhancer v0.2 with React v0.14+ soon and the warnings will gone.

**Comment by tgienger** [comment]
Just checking:
```
Warning: ReactDOMComponent: Do not access .props of a DOM node; instead, recreate the props as `render` did originally or read the DOM properties/attributes directly from this node (e.g., this.refs.box.className). This DOM node was rendered by `GSAPEnhancer`.
```
also be fixed?

**Comment by azazdeaz** [comment]
Hi @tgienger!

It should be already fixed since v0.1.8. Which version are you using?

**Comment by tgienger** [comment]
right, my bad. thanks @azazdeaz .

**Comment by azazdeaz** [comment]
closed with 82bc2309be4cda35ff0aa993c0e0b53cdd4717a4
